build-and-test.ps1

```diff
@@ -22,6 +39,7 @@ $manifest = Get-Content "manifest.json" | ConvertFrom-Json
 $manifestRepo = $manifest.Repos[0]
 $activeOS = docker version -f "{{ .Server.Os }}"
 $builtTags = @()
+Invoke-CleanupDocker
```

Comment:
I think cleaning up the environment should be done as soon as possible - e.g. line 30.